ingestion_tools/scripts/importers/annotation.py

```diff
@@ -469,8 +541,25 @@ def import_metadata(self):
         print(filename)
         self.annotation_metadata.write_metadata(filename, self.local_metadata)
 
+    def import_neuroglancer_component(self):
```

Comment:
I'd prefer this stuff to be refactored into a NeuroglancerPrecomputeImporter when we get a chance.